src/cards/AlliesCard/AlliesCard.jsx

```diff
@@ -1,22 +1,56 @@
-import React from "react";
+import React, { useState } from "react";
 import "./AlliesCard.scss";
 import image from "./allies_headshot.JPG";
 
 function AlliesCard() {
+  const [darkMode, setDarkMode] = useState(false);
+
+  function toggleDarkMode() {
+    if (darkMode === false) {
+      setDarkMode(true);
+    } else {
+      setDarkMode(false);
+    }
+  }
+
   return (
-    <div className="cardContainer">
-      <div className="card">
+    <div className={`${darkMode ? "dark" : "light"}AllieCardContainer`}>
+      <div className="alliecard">
         <img
           className="allieImageContainer"
           src={image}
           alt="Smiling tall white woman with long blonde hair wearing glasses and a red and deep blue floral print blouse sits in front of a white brick wall"
         />
       </div>
 
-      <div className="card">
+      <div className="alliecard">
         <h3>Allie LeFever</h3>
         <p>Software Engineer</p>
-        <a href="https://www.linkedin.com/in/allie-lefever/">LinkedIn</a>
+        <button className="allieLinkButton">
+          <a
+            className="allieLink"
+            href="https://www.linkedin.com/in/allie-lefever/"
+          >
+            LinkedIn
+          </a>
+        </button>
+        <button className="allieLinkButton">
+          <a className="allieLink" href="https://github.com/alefever39">
+            GitHub
+          </a>
+        </button>
+      </div>
+
+      <div></div>
+      <div className="allieDarkButtonContainer">
+        <button
+          onClick={toggleDarkMode}
+          className={`allieDarkButton ${
+            darkMode ? "dark" : "light"
+          }AllieDarkButton`}
+        >
+          Dark Mode
```

Comment:
noticed a couple of things with the dark mode button:

- (not blocking) What would you think of having the button display "Light Mode" when dark is active and "Dark Mode" when light is active? 
- the text "Dark Mode" is not appearing within the button for me(see screenshot), is this something you see locally? My first thought is it could be a screen size/responsive issue. 
- When dark mode is active and the dark mode button is hovered, the white text on the light blue background could be hard to read and does not meet contrast guidelines. Could the text switch to black or use a darker shade of blue? Here is a quick contrast checker: https://webaim.org/resources/contrastchecker/ 
  
<img width="714" alt="Screen Shot 2022-10-27 at 4 04 27 PM" src="https://user-images.githubusercontent.com/63808889/198406878-00f36410-72ae-4181-bc34-23c41e8747bb.png">

Reply:
Yeah, I can make these changes now. Thanks for the feedback!

Reply:
I just added these changes now, let me know if you think anything else should change!